src/Application/Regulation/View/PeriodView.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Regulation\View;
+
+class PeriodView
+{
+    public function __construct(
+        public readonly array $applicableDays,
```

Comment:
La variable semble plutôt contenir des `dayRanges`

```suggestion
        public readonly array $dayRanges,
```